.circleci/config.yml

```diff
@@ -6,17 +6,69 @@ jobs:
 
   build-python36:
     docker:
-      - image: ocrd/core
+      - image: ubuntu:18.04
     steps:
-      - run: apt-get update && apt-get install -y --no-install-recommends make git curl
+      - run: apt-get update
+      - run: apt-get install -y software-properties-common
+      - run: add-apt-repository -y ppa:deadsnakes/ppa
+      - run: apt-get update && apt-get install -y --no-install-recommends make git curl python3.6
       - checkout
       - run: make deps-ubuntu
-      - run: make install
-      - run: make test-cli
-      - run: make coverage
+      - run: make install PYTHON=python3.6
+      - run: make test-cli PYTHON=python3.6
+      - run: make coverage PYTHON=python3.6
+      - codecov/upload
+
+  build-python37:
```

Comment:
Could this be made more elegant in Circle CI by just setting the `PYTHON` variable differently without repeating the same step definitions each time?

Reply:
CircleCI probably has some mechanism for build matrices, I'll look into it. While I agree that copy/pasting isn't very elegant, we (hopefully) don't have to change the configuration all that often so the redundancy isn't that big of a problem IMHO.

Reply:
https://github.com/OCR-D/ocrd_tesserocr/pull/176/commits/a14d73c8b840356ea01026268c9ff1385a7b17e9